src/syspurpose/files.py

```diff
@@ -206,9 +205,9 @@ def write(self, fp=None):
             write_to_file_utf8(fp, self.contents)
 
     @classmethod
-    def read(cls, path, raise_on_error=False):
+    def read(cls, path: str, raise_on_error: bool = False) -> "SyspurposeStore":
```

Comment:
I aware of `annotations` from `__future__`, but I prefer this solution, because using `annotations` could have some unexpected side effect. We can use it later, but not now.